src/mongoose_scram.erl

```diff
@@ -35,13 +35,14 @@
                          Salt :: binary(), Iterations :: non_neg_integer() }.
 
 -type scram_map() ::
-    #{salt := binary(),
-      iteration_count := non_neg_integer(),
+    #{iteration_count := non_neg_integer(),
       sha_key() := server_and_stored_key_type()}.
 
 -type sha_key() :: sha | sha224 | sha256 | sha384 | sha512.
 
--type server_and_stored_key_type() :: #{server_key := binary(), stored_key := binary()}.
+-type server_and_stored_key_type() :: #{salt      := binary(),
+                                       server_key := binary(),
+                                       stored_key := binary()}.
```

Comment:
Looks like we miss one space of indentation to align with the begining of `salt` in the line above. Or is it just how GtiHub displayed this change?

Reply:
I missed this one. Good catch, will change that and push again.